### DISCUSSION TITLE
geojsonLayer does not parse json file, no geometry

### DISCUSSION THREAD

**Discussion by qiaoxin136** [visgl/deck.gl#6489] [Q&A]
i set up a codesandbox to display a line geojson file. the browser says "GeoJSON does not have geometry". I used https://geojsonlint.com/ to check the file and it displays correctly. Can anyone see this before? any recommendations? Thanks a lot. 

https://codesandbox.io/s/empty-snow-ycmnz?file=/app.js

https://hollywood-google-map.s3.amazonaws.com/pipe.json

**Comment by Pessimistress** [comment]
As the error says, there is a feature in the GeoJSON that does not have geometry.

![image](https://user-images.githubusercontent.com/2059298/145303618-970f43e9-8d01-43c0-90ee-34c2da1596e9.png)
